src/Util/AdminObjectAclUserManager.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Util;
+
+/**
+ * @author Mathieu Petrini <mathieupetrini@gmail.com>
+ */
+interface AdminObjectAclUserManager
```

Comment:
Is this interface needed?

Reply:
yes, for DI in CRUDController

Reply:
But no one is implementing this new interface

Reply:
Yes, interface must be defined by final users.

It was to maintain this functionality with DI (https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Controller/CRUDController.php#L1346)

Reply:
Before, users should create a class. Now a class with this interface implemented.